var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -184,10 +186,13 @@ def cmake_args(self):
 
         # Currently FFTW_HOME and LIBXML2_HOME are used by CMake.
         # Any CMake warnings about other variables are benign.
-        xml2_prefix = spec['libxml2'].prefix
-        args.append('-DLIBXML2_HOME={0}'.format(xml2_prefix))
-        args.append('-DLibxml2_INCLUDE_DIRS={0}'.format(xml2_prefix.include))
-        args.append('-DLibxml2_LIBRARY_DIRS={0}'.format(xml2_prefix.lib))
+        # Starting with QMCPACK 3.8.0, CMake uses the builtin find(libxml2)
+        # function
+        if spec.satisfies('@:3.7.0'):
```

Comment:
Is the `libxml2` dependency no longer needed or is CMake smart enough to find it on its own now?

Reply:
libxml2 is still needed. Starting with QMCPACK 3.8.0, we use CMake's built-in function. Prior to 3.8.0 -- I am not sure, for some reason we needed those variables. Most likely the functionality was in CMake and just never used it.